src/extern.jl

```diff
@@ -2,21 +2,23 @@
 
 # Main function
 """
-    heppcat(Y,k,iters;init=homppca(Y,k))
+    heppcat(Y, k, iters; init=homppca(Y,k), vknown=false, eps=0)
 
 Estimate probabilistic PCA model for noise that is heteroscedastic across samples.
 
-Inputs are:
+# Inputs:
 + `Y` : list of matrices (each column is a sample)
 + `k` : number of factors
 + `iters` : number of iterations to run
 + `init`  : initial model (will be modified in-place)
++ `vknown::Bool` : variances are known (do not update) default `false`
++ 'eps::Real' : epsilon to avoid variances going towards zero, useful if normal method fails, default = 0
```

Comment:
```suggestion
+ 'vmin::Real' : lower bound on variances; useful if normal method fails, default = 0
```
`eps` is a Base function in Julia already, and also having a more descriptive name might be helpful.

Reply:
Agreed. I think we should use `varfloor` for now and describe its use case a bit more.
```suggestion
+ 'varfloor' : lower bound on variances; useful if variance estimates are degenerating to 0, default = 0
```